content/pl/docs/tasks/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Zadania
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+W tej części dokumentacji Kubernetesa znajdują się opisy
+sposobu realizacji różnych zadań. Przedstawione są one zazwyczaj jako
+krótka sekwencja kilku kroków związanych z pojedynczym zadaniem.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Graficzny interfejs użytkownika _(Dashboard)_
+
+Instalacja i użycie graficznego interfejsu użytkownika z poziomu przeglądarki do zarządzania aplikacjami w kontenerach uruchomionymi na klastrze Kubernetes.
+
+## Jak używać polecenia kubectl
+
+Instalacja i konfiguracja polecenia `kubectl` do bezpośredniego zarządzania klastrami Kubernetes.
+
+## Konfigurowanie podów i kontenerów
+
+Najpopularniejsze czynności związane z konfiguracją podów i kontenerów.
+
+## Uruchamianie aplikacji
+
+Standardowe metody zarządzania aplikacjami, m. in. prowadzenie stopniowych aktualizacji _(rolling updates)_, przekazywanie konfiguracji oraz skalowanie horyzontalne podów.
+
+## Uruchamianie zadań
+
+Uruchamianie zadań w trybie równoległym.
+
+## Dostęp do aplikacji na klastrze
+
+Rozkładanie obciążenia i przekierowywanie ruchu sieciowego, konfigurowanie firewalla i usług DNS w celu zapewnienia dostępu do aplikacji w klastrze.
+
+## Monitoring, rejestracja i znajdowanie błędów
+
+Konfigurowanie monitoringu oraz rejestrowanie zdarzeń i komunikatów, pomagające rozwiązywać problemy związane z pracą klastra lub aplikacji uruchamianych w kontenerach.
+
+## Dostęp do API Kubernetes
+
+Różne metody bezpośredniego dostępu do API Kubernetes.
+
+## Używanie TLS
+
+Konfigurowanie aplikacji w taki sposób, aby korzystała i ufała łańcuchowi certyfikatów wydawanych przez główne centrum autoryzacji (CA) klastra.
```

Comment:
Nit picking: `centrum autoryzacji` sounds a bit weird, AFAIR it's Certificate Authority? so it should be rather `Urząd Certyfikacji`.